sktime/forecasting/trend/_pwl_trend_forecaster.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements a piecewise linear trend forecaster by wrapping fbprophet."""
+
+__author__ = ["sbuse"]
+
+import pandas as pd
+
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.base.adapters import _ProphetAdapter
+
+
+class PiecewiseLinearTrendForecaster(_ProphetAdapter):
+    """
+    Forecast time series data with a piecwise linear trend.
+
+    The forecaster uses Facebook's prophet algorithm [1]_ and extracts the piecewise
+    linear trend from it. Only hyper-parameters relevant for the trend modelling are
+    exposed via the constructor.
+
+    Data can be passed in one of the sktime compatible formats,
+    naming a column `ds` such as in the prophet package is not necessary.
+
+    Unlike vanilla `prophet`, also supports integer/range and period index:
+    * integer/range index is interpreted as days since Jan 1, 2000
```

Comment:
This implementation will always automatically add two additive seasonal components (`sp=7` and `sp=365`) to the underlying model, even for if the underlying data is non-daily (since we're now passing the daily assumption to Prophet). And since the user can only pass trend-related hyperparameters to the constructor, there is no way to disable this behaviour. I believe this is a very unintuitive default behaviour to have in a class that appears to be a simple generic piecewise linear trend forecaster. As I mentioned in [this comment](https://github.com/sktime/sktime/pull/5592#issuecomment-1857745836), I belive that disabling all seasonal components in the underlying Prophet model (`[daily|weekly|yearly]_seasonality=False`) would solve this.

Reply:
so, let's do that? I did not expect seasonal components.

Reply:
I renamed the estimator and updated the docstring to clarifiy what happens with seasonality